terraform/modules/hub/files/tasks/hub-policy.json

```diff
@@ -89,6 +89,10 @@
       {
         "Name": "JAVA_OPTS",
         "Value": "-Dservice.name=policy ${jvm_options} -XX:+HeapDumpOnOutOfMemoryError -Dnetworkaddress.cache.ttl=5 -Dnetworkaddress.cache.negative.ttl=5"
+      },
+      {
+        "Name": "LOG_LEVEL",
+        "Value": "${service_log_level}"
```

Comment:
Can we add the definition of service_log_level variable to  terraform/modules/hub/variables.tf.

Reply:
Can we add service_log_level to vars in template_file in policy, saml-engine, etc. For example, 

```
data "template_file" "policy_task_def" {
  template = file("${path.module}/files/tasks/hub-policy.json")

  vars = {
    ...
    service_log_level                   = var.service_log_level
  }
}
```

Reply:
Ok, thanks. Terraform..